### PR TITLE
DEVOPS1003-dockerize go

### PR DESCRIPTION
**Pull request by mxq7854687**
https://trello.com/c/6SCR5Cc0/61-devops1003-1-dockerize-go-for-employee-system

run `docker-compose up` 
image
example for working logs.

run `docker-compose up -d` for detached mode.
run `docker-compose down` to stop and remove the container 

The deployment pipeline would be added in the another PR

### CODE REVIEW

File: wait-for.sh [hunk 1]
```diff
@@ -0,0 +1,191 @@
+#!/bin/sh
+
+# The MIT License (MIT)
```
Comment (cheungjaidev) [inline wait-for.sh:3]: Did you generate the MIT License

**Comment by cheungjaidev** [comment]
Do you think you can go for a demo on Sunday?

File: wait-for.sh (hunk 1 above)
Comment (mxq7854687) [reply wait-for.sh:3]: not yet

**Comment by mxq7854687** [comment]
yes i can do a quick demo for that

**Comment by cheungjaidev** [comment]
> yes i can do a quick demo for that

Merge it yourself if finished. I am still waiting for the pull request from @yutooo5225

**Pull request merged**